test-fixtures/update-rust-component-version.py

```diff
@@ -42,6 +44,45 @@ def read_rust_components_tag_version(SPM_PACKAGE):
         logging.info(f"Unexpected error: {e}")
     return None, None
 
+def focus_read_rust_components_tag_version():
+    # Read Package file to find the current rust-component version
+    f = open(FOCUS_SPM_PACKAGE)
+    data = json.load(f)
+    pin = data["object"]
+
+    for i in pin["pins"]:
+        if i["package"] == "MozillaRustComponentsSwift":
+            # return the json with the RustComponent info
+            json_new_version = i["state"]
+    f.close()
+
+    # Return the current version and commit
+    return json_new_version["version"], json_new_version["revision"]
+
+def focus_read_project_min_version():
```

Comment:
Same question for this one—I wonder if you can reuse `read_project_min_version()` for both Focus and Firefox iOS, instead of parsing the Focus Xcode project separately?

Maybe with something like this:

```python
# From http://github.com/mozilla-mobile/firefox-ios/blob/main/focus-ios/Blockzilla.xcodeproj/project.pbxproj#L6547-L6554
FOCUS_RUST_COMPONENTS_ID = "45E8FFE52828DE4A0027A8F5"

focus_current_min_version = read_project_min_version(FOCUS_PROJECT, FOCUS_RUST_COMPONENTS_ID)
```